eclipse/sw360
# fix(upgradeVersion):Upgrade version.

[//]: # (Copyright Bosch.IO GmbH 2020)
[//]: # (This program and the accompanying materials are made)
[//]: # (available under the terms of the Eclipse Public License 2.0)
[//]: # (which is available at https://www.eclipse.org/legal/epl-2.0/)
[//]: # (SPDX-License-Identifier: EPL-2.0)

> **It is recommended to upgrade some existing software version which sw360 is currently using as part of the technical debt.**
> * Which issue is this pull request belonging to and how is it solving it? (#1202 #1203 #1204 #1205 #1206 #1207)
> * Did you add or update any new dependencies that are required for your change? No

Issue: closes (#1202 #1203 #1204 #1205 #1206 #1207)

### Suggest Reviewer
> You can suggest reviewers here with an @mention.

### How To Test?
> How should these changes be tested by the reviewer? 

- First install the Thrift 0.14 
- Do a mvn clean install without skipping any test cases 
- Test the application end to end from the UI and REST end.
> Have you implemented any additional tests? No

### Checklist
Must:

Signed-off-by: ravi110336 <kumar.ravindra@siemens.com>
